.github/workflows/ci.yaml

```diff
@@ -0,0 +1,42 @@
+name: CI
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  unittest:
+    name: unit tests
+    runs-on: ubuntu-22.04
```

Comment:
nit: maybe use latest here, since you're not doing anything specific like "apt install"ing.